helper/runner.go

```diff
@@ -473,8 +473,7 @@ func simpleDecodeModuleCallBlock(block *hcl.Block) (*terraform.ModuleCall, hcl.D
 		SourceAddrRange: sourceAddrRange,
 		SourceSet:       !sourceAddrRange.Empty(),
 
-		Config:      remain,
-		ConfigRange: block.DefRange,
```

Comment:
`ConfigRange` does not exist in the original terraform configuration and is used only for intermediate communication via RPC. It is not intended to be public, so we will remove it at this time.